src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  public Random generator = new Random();
```

Comment:
This random generator has been used locally only. This should be private.

Reply:
Okay, I'll make it private